src/postgres/event-handlers/RootPostDeleted.ts

```diff
@@ -0,0 +1,12 @@
+import { PostId } from '@subsocial/types/substrate/interfaces/subsocial';
+import { deleteNotificationsAboutPost } from '../delete-activity';
+import { deletePostFollower } from '../delete-follower';
+import { SubstrateEvent, EventHandlerFn } from '../../substrate/types';
+
+export const onRootPostDeleted: EventHandlerFn = async (eventAction: SubstrateEvent) => {
+  const { data } = eventAction;
+  const follower = data[0].toString();
+  const following = data[1] as PostId;
```

Comment:
Fix this copypasta on all post events handlers:

```typescript
const { data } = eventAction;
const follower = data[0].toString();
const postId = data[1] as PostId;
```

it should be something like this everywhere:

```typescript
const { author, postId } = parsePostEvent(eventAction);
```